docs/articles/intro/index.rst

```diff
@@ -0,0 +1,66 @@
+Introduction
+************
+
+This is the Software Factory Blueprint.
+
+How to use this blueprint
+=========================
+The blueprint is not intended to be particularly useful on its own, rather it is a collection of
+generic descriptions, processes, how-tos et cetera.
+
+The typical user of this blueprint is someone who wants to run a project where a Software Factory is
+part of the project. The blueprint can then be used as a base and then be extended with
+project-specific info, or one can just pick and choose the relevant parts from the blueprint.
+
+.. only:: blueprint
+
+    If you are reading this, it means you are reading the blueprint and not a deployment, meaning
+    that some links or names may look weird. Don't be alarmed, this is (mostly) by design.
+
+A nice way of using this blueprint in a project-specific setting is to use it as a *git submodule*
```

Comment:
In general I think we should write how to do this, not recommend one way of doing it. The difference is often small, but the docs need to read consistently throughout, and this is the manual of how to actually do things. When a rationale of why we don something in a specific way or when it's relevant to compare different approaches that could be separated from the other text.